internal/pkg/show/config/plugins/jenkins-pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+tools:
+# name of the tool
+- name: jenkins-pipeline
+  # id of the tool instance
+  instanceID: default
+  # format: name.instanceID; If specified, dtm will make sure the dependency is applied first before handling this tool.
+  dependsOn: [ ]
+  # options for the plugin
+  options:
+    jenkinsURL: JENKINSURL
```

Comment:
How about set default/sample values here? Like: http://127.0.0.1:8080. These could be equal to default values it they have default values.